countdown.js

```diff
@@ -1,5 +1,20 @@
-function countdown(seconds){
-  // ...
-}
+//Exercise 2 - Countdown 
+function countdown(seconds) {
+  var timer = parseInt(seconds);
+  var delay = 1000;
+  try {
+    var startTimer = setInterval(function() {
+      console.log(timer);
+      timer--;
+      if (timer == 0) {
+        clearInterval(startTimer);
+      }
+    }, delay);
 
+  } catch (e) {}
```

Comment:
We talked about it in class, but try to avoid `try/catch` unless there is a case where a failure is possible, and there isn't an easy way to handle it another way.

http://programmers.stackexchange.com/a/144328/45713